src/config/mod.rs

```diff
@@ -1847,6 +1859,18 @@ impl Config {
         self.colors.cursor.cursor.map(|_| Color::Named(NamedColor::Cursor))
     }
 
+    /// Selected text color.
+    #[inline]
+    pub fn selection_text_color(&self) -> Option<Color> {
+        self.colors.selection.text.map(|_| Color::Named(NamedColor::SelectionText))
+    }
+
+    /// Selected text background color.
+    #[inline]
+    pub fn selection_selection_color(&self) -> Option<Color> {
```

Comment:
This name is pretty bad, I think `selection_background_color` would be the right choice here.